spec/factories/efile_submissions.rb

```diff
@@ -35,5 +35,13 @@
         end
       end
     end
+
+    trait :with_errors do
+      after :create do |submission|
+        file = Pathname.new(File.join(Rails.root, "/spec/fixtures/files", "irs_acknowledgement_rejection.xml")).read
+        raw_response = Nokogiri::XML(file).to_xml
```

Comment:
Since you're storing the raw XML contents as a string in the `jsonb` Postgres field, I think it's more accurate to never parse the data with Nokogiri. Consider instead:

```
        raw_xml = File.read(File.join(Rails.root, "spec/fixtures/files", "irs_acknowledgement_rejection.xml"))
        submission.efile_submission_transitions.last.update(metadata: { raw_response: raw_xml })
```

Also, to my eyes, it's more common to do `spec/fixtures/files` as the subdirectory path, not `/spec/fixtures/files` as the subdirectory path; I use a leading slash to indicate a pathname is absolute, so I found leading slash confusing.

Not a blocker.